specs/deliverly_server_SUITE.erl

```diff
@@ -85,6 +94,56 @@ auth_failed(_) ->
   1 = Size,
   ok.
 
+auth_token_success(_) ->
+  Token = proplists:get_value(token, de_auth_token:request_token(#{})),
+  Res = deliverly_server:auth_client(#de_client{socket=1, app = default}, [{<<"token">>, Token}]),
```

Comment:
Line 99 is too long:   Res = deliverly_server:auth_client(#de_client{socket=1, app = default}, [{<<"token">>, Token}]),.